actions/linting/action.yml

```diff
@@ -23,6 +23,7 @@ runs:
         path: |-
           ~/.ansible
           ~/.cache/pre-commit
+        # yamllint disable-line rule:quoted-strings
         key: "\
           cache-\
           ${{ hashFiles(inputs.ansible-roles-config) }}-\
```

Comment:
Does this mean you don't like my suggested approach?

Reply:
oh sorry I didn't register what you had done in the link you shared - I thought you were suggesting to move the comment to the line above. Will fix now

Reply:
Cool. Wasn't sure if it was missed, or you didn't agree 🤷